linkage/demo_notebooks/2022_11_09a_prl_splink_test_part_2.ipynb

```diff
@@ -0,0 +1,1858 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        df_predictions, threshold_match_probability=0.5
</pre><p>I wonder how this value is chosen in practice? With a ground truth column like we have here, you can create a precision-recall chart as below, but it's not clear how to do this without ground truth. I imagine the thing I'd really want to tune as a decision maker is precision and recall, not the match probability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>